modules/doc/content/newsletter/2020_12.md

```diff
@@ -2,4 +2,16 @@
 
 ## Migration to Github Discussions
 
-The MOOSE development team decided to move all new questions to a [Github discussion forum](https://github.com/idaholab/moose/discussions).
+The MOOSE development team decided to move all new questions to a [Github discussion
+forum](https://github.com/idaholab/moose/discussions).
```

Comment:
This link doesn't seem to want to render in the website preview (https://mooseframework.inl.gov/docs/PRs/16457/site/newsletter/2020_12.html), so I think you might want this all on a single line.